generators/client/templates/react/src/main/webapp/app/modules/administration/configuration/configuration.tsx.ejs

```diff
@@ -54,109 +53,109 @@ export class ConfigurationPage extends React.Component<IConfigurationPageProps,
       this.props.getConfigurations();
       this.props.getEnv();
     }
-  }
+  };
 
   setFilter = evt => {
     this.setState({
       filter: evt.target.value
     });
-  }
+  };
 
-  filterFn = prefix => prefix.toUpperCase().includes(this.state.filter.toUpperCase());
+  envFilterFn = configProp => configProp.toUpperCase().includes(this.state.filter.toUpperCase());
+  propsFilterFn = configProp => configProp.prefix.toUpperCase().includes(this.state.filter.toUpperCase());
 
-  sortPrefix = (configProps, rev) => {
-    const { configuration } = this.props;
-    if (configuration && configuration.configProps) {
-      const listSorted = sortBy(configProps, [ prop => configuration.configProps[prop].prefix ]);
-      return rev ? reverse(listSorted) : listSorted;
-    }
-  }
+  reverseIf = (condition, data) => condition ? data.reverse() : data;
 
-  sortProperties = (configProps, rev) => {
-    const { configuration } = this.props;
-    if (configuration && configuration.configProps) {
-      const listSorted = sortBy(configProps);
-      return rev ? reverse(listSorted) : listSorted;
-    }
-  }
+  sortPrefix = (configProps, rev) => this.reverseIf(rev, Object.values(configProps).sort((a, b) => a['prefix'].localeCompare(b['prefix'])));
+
+  sortProperties = (configProps, rev) => this.reverseIf(rev, Object.keys(configProps).sort());
```

Comment:
Yes, I dont think its a good idea to sort Object keys as it might cause a prefix to show wrong property. I would even suggest to get rid of sorting here and keep only filtering as that is what matters, I dont see much use of sorting these

Reply:
If it is required then best way would be to transform the object to array with both key and value first and then sort the array

Reply:
Alright, I'll remove it then.